sources/packages/web/src/components/layouts/FullPageContainer.vue

```diff
@@ -1,9 +1,53 @@
 <template>
-  <v-container>
-    <v-row justify="center">
-      <v-card class="p-4 w-100 full-page-container-size">
-        <slot></slot>
-      </v-card>
-    </v-row>
+  <v-container :fluid="fullWidth">
+    <header class="mb-4">
+      <slot name="header"></slot>
+    </header>
+    <slot name="alerts"></slot>
+    <template v-if="layoutTemplate === LayoutTemplates.CenteredCard">
+      <v-row justify="center">
+        <v-card class="p-4 w-100" :class="widthClass">
+          <slot></slot>
+        </v-card>
+      </v-row>
+    </template>
+    <template v-else-if="layoutTemplate === LayoutTemplates.Centered">
+      <v-row justify="center">
+        <div class="w-100" :class="widthClass">
+          <slot></slot>
+        </div>
+      </v-row>
+    </template>
   </v-container>
 </template>
+
+<script lang="ts">
+import { computed } from "vue";
+
+enum LayoutTemplates {
+  Centered = "Centered",
```

Comment:
I know for enum we discided to use PascalCase for enum, but since these values are related to the layout API property, can we use hyphen-case like `centered and centered-card`. just a thought to know the opinion of the dev. 
Generally in frameworks like vuetify, we use hyphen-case 
![image](https://user-images.githubusercontent.com/77353155/169163711-fe63334b-d18f-4b8f-ac08-facf3fe9fd6a.png)

Reply:
I believe that would be better, I will change it.

Reply:
Enum values updated.